crates/fuel-core/src/database/database_description/compression.rs

```diff
@@ -0,0 +1,28 @@
+use crate::database::database_description::DatabaseDescription;
+use fuel_core_compression_service::storage::column::CompressionColumn;
+use fuel_core_storage::merkle::column::MerkleizedColumn;
+use fuel_core_types::fuel_types::BlockHeight;
+
+#[derive(Clone, Copy, Debug)]
+pub struct CompressionDatabase;
+
+impl DatabaseDescription for CompressionDatabase {
+    type Column = MerkleizedColumn<CompressionColumn>;
+    type Height = BlockHeight;
+
+    fn version() -> u32 {
+        0
+    }
+
+    fn name() -> String {
+        "compression".to_string()
+    }
+
+    fn metadata_column() -> Self::Column {
+        Self::Column::MerkleMetadataColumn
+    }
```

Comment:
I think we should have a dedicated metadata column. I had problems trying to use the `MerkleMetadataColumn` as the metadata column in the state root service. This is what I had to do to fix it: https://github.com/FuelLabs/fuel-proof-system/pull/42/commits/41c6fb0ef46c016c9261f0f736798b107a4c23a0

Reply:
Though, since you don't use the metadata table but instead the blocks for the `Modifiable` implementation you won't hit the same bug I did. So I suppose the metadata column will be unused, but it would be safer from other potential bugs to have a dedicated metadata column even if it's empty initially.

Reply:
Sure, though perhaps it's then more correct to set `CompressedBlocks` as the metadata table here? :thinking: 

Anyway I agree I'm not worried about this being a production bug, since tests would catch it. I'll approve and let you decide if you want to change the metadata table or not.